frontend/src/features/workspace/WorkspaceService.ts

```diff
@@ -94,8 +95,8 @@ export const createEmailModeForm = async (
   )
 }
 
-export const createStorageModeForm = async (
-  body: CreateStorageFormBodyDto,
+export const createStorageModeOrMultirespondentForm = async (
```

Comment:
nit: `createEncryptedForm`?

Reply:
i considered having this naming change where we have a mini hierarchy
```
                forms
              /       \
  not encrypted     encrypted
          |          /    \
        email    storage   MRF
```
But I think since a lot of naming conventions in the submissions directory mix up storage <-> encrypt, it would become pretty confusing if we do the renaming in some places but not others. I think we can refactor this another time with renamings, where even the whole `encrypt-submission` folder should become `storage-submission`, and then we can move to this naming scheme.